packages/ingest/HttpProviderClient.js

```diff
@@ -42,7 +42,15 @@ class HttpProviderClient {
   list(path) {
     validateHost(this.host);
 
-    const pattern = /<a href="([^>]*)">[^<]+<\/a>/;
+    // Make pattern case-insensitive and return all matches
+    // instead of just first one
+    const matchLinksPattern = /<a href="([^>]*)">[^<]+<\/a>/ig;
```

Comment:
It still makes me uncomfortable to rely on regular expressions for finding links in source code, but I don't have any better ideas

Reply:
Yeah, this isn't super awesome.  It feels like the sort of thing we should do by pulling in the page with a dom parser and searching the document (something like jsdom) using those sorts of tools rather than rolling a regexp. 

That's *probably* well outside the scope of this PR, however.  If that suggestion sounds interesting I'd be up for a conversation or a IP/TD ticket.

Reply:
I might take a look and see how much it work it is to implement JSDOM unless you strongly object

Reply:
I strongly don't object.

Reply:
I had to abandon this idea. JSDOM just does not play nicely with webpack